<ui version="4.0">
  <class>TGo4HisDrawOptions</class>
  <widget class="QWidget">
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>736</width>
        <height>29</height>
      </rect>
    </property>
    <property name="minimumSize" >
      <size>
        <width>0</width>
        <height>0</height>
      </size>
    </property>
    <property name="windowTitle" >
      <string>Histogram Draw Options</string>
    </property>
    <layout class="QGridLayout" >
      <property name="margin" >
        <number>0</number>
      </property>
      <property name="spacing" >
        <number>0</number>
      </property>
      <item row="0" column="0" >
        <widget class="QComboBox" name="DrawOption" />
      </item>
      <item row="0" column="1" >
        <widget class="QComboBox" name="ErrorBars" />
      </item>
      <item row="0" column="2" >
        <widget class="QComboBox" name="Coordinates" />
      </item>
      <item row="0" column="3" >
        <widget class="QCheckBox" name="LogX" >
          <property name="text" >
            <string>LogX</string>
          </property>
        </widget>
      </item>
      <item row="0" column="4" >
        <widget class="QCheckBox" name="LogY" >
          <property name="text" >
            <string>LogY</string>
          </property>
        </widget>
	</item>
      <item row="0" column="5" >
        <widget class="QCheckBox" name="LogZ" >
	   <property name="text" >
              <string>LogZ</string>
            </property>
        </widget>
      </item>
      <item row="0" column="6" >
        <widget class="QCheckBox" name="AutoScaleBox" >
          <property name="text" >
            <string>Auto Scale</string>
          </property>
        </widget>
      </item>
      <item row="0" column="7" >
        <widget class="QToolButton" name="LineColor" >
          <property name="sizePolicy" >
            <sizepolicy>
              <hsizetype>0</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="maximumSize" >
            <size>
              <width>30</width>
              <height>30</height>
            </size>
          </property>
          <property name="autoRaise" >
            <bool>true</bool>
          </property>
          <property name="icon" >
            <iconset>:/icons/changelincol.png</iconset>
          </property>
          <property name="toolTip" stdset="0" >
            <string>Change line color of selected object.</string>
          </property>
        </widget>
      </item>
      <item row="0" column="8" >
        <widget class="QToolButton" name="FillColor" >
          <property name="sizePolicy" >
            <sizepolicy>
              <hsizetype>0</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="text" >
            <string/>
          </property>
          <property name="maximumSize" >
            <size>
              <width>30</width>
              <height>30</height>
            </size>
          </property>
          <property name="icon" >
            <pixmap>:/icons/changefillcol.png</pixmap>
          </property>
          <property name="autoRaise" >
            <bool>true</bool>
          </property>
          <property name="icon" >
            <iconset>:/icons/changefillcol.png</iconset>
          </property>
          <property name="toolTip" stdset="0" >
            <string>Change fill color for selected object.</string>
          </property>
        </widget>
      </item>
      <item row="0" column="9" >
        <widget class="QToolButton" name="MarkerColor" >
          <property name="sizePolicy" >
            <sizepolicy>
              <hsizetype>0</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="text" >
            <string/>
          </property>
          <property name="maximumSize" >
            <size>
              <width>30</width>
              <height>30</height>
            </size>
          </property>
          <property name="icon" >
            <pixmap>:/icons/changemarcol.png</pixmap>
          </property>
          <property name="autoRaise" >
            <bool>true</bool>
          </property>
          <property name="icon" >
            <iconset>:/icons/changemarcol.png</iconset>
          </property>
          <property name="toolTip" stdset="0" >
            <string>Change marker color of selected object.</string>
          </property>
        </widget>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="0" margin="0" />
</ui>
